README.md

```diff
@@ -71,7 +71,7 @@ For protobuf-based codegen integrated with the Maven build system, you can use
         -->
         <protocArtifact>com.google.protobuf:protoc:3.0.0-alpha-2:exe:${os.detected.classifier}</protocArtifact>
```

Comment:
The protobuf versions need to get updated to 3.0.0-alpha-3.1 as well (to align with what 0.8.0 used).